content/docs/user-guide/settings/issue-az-console-commands.md

```diff
@@ -167,10 +167,12 @@ Because the `deferredCommand2` cannot run immediately, it runs after all of the
-1. Run Console commands specified on the command line. Specified using command option notation `-<console command name> <args>`). For example, `-loadlevel <levelname>`.
-1. Activate System Components in Gems.
+1. Create the [Settings Registry](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#LL466C16-L466C16).
+1. Create the [AZ Console](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#L466C16-L475).
+2. Create the [Archive FileIO](https://github.com/o3de/o3de/blob/bc24a8fefb5438f4782f34caa0f9ecb3504d04c3/Code/Framework/AzFramework/AzFramework/Application/Application.cpp#L106-L137)
+3. Merge all Settings Registry files (`.setreg`/`.setregpatch`) to the [registry](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#L762C1-L788) and attempt to invoke any Console commands underneath the monitored [Settings Registry Console Command](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Console/Console.cpp#L527-L605) keys. Any commands that cannot run are added to the AZ Console [deferred command queue](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Console/Console.cpp#L103C8-L125).
+4. Load Dynamic Modules (Gem modules). Sends the `GemsLoaded` lifecycle event. For each loaded Gem, deferred commands attempt to [execute](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Module/Module.h#L99-L114). Any that succeed are [removed](https://github.com/o3de/o3de/blob/bc24a8fefb5438f4782f34caa0f9ecb3504d04c3/Code/Framework/AzCore/AzCore/Console/Console.cpp#L196-L203) from the deferred command queue.
+5. Attempt to run Console commands in the [user.cfg](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#L843-L849) from the Projects asset cache folder (usually `<project-root>/Cache/<platform>`).
+6. Run Console commands specified on the [command line](https://github.com/o3de/o3de/blob/7d69221a92f1eb830573ade15cb327b5ede6346c/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#L851-L852). Specified using command option notation `-<console command name> <args>`). For example, `-loadlevel <levelname>`.
+7. Set the FileIO aliases `@alias@` for [all active gems](https://github.com/o3de/o3de/blob/bc24a8fefb5438f4782f34caa0f9ecb3504d04c3/Code/Framework/AzFramework/AzFramework/Application/Application.cpp#L199-L205)
+8. Activate System Components in [Gems](https://github.com/o3de/o3de/blob/bc24a8fefb5438f4782f34caa0f9ecb3504d04c3/Code/Framework/AzCore/AzCore/Module/ModuleManager.cpp#L633-L783).
```

Comment:
Nit: Everything comes out ok as-is, but stylistically it would be nice to either number them all as 1 or all as sequential, instead of 1, 1, 2, 3, 4...

Reply:
Ahh that is the markdown plugin I have in VSCode, that auto re-numbered the values.
I'll fix these values back to 1.